azurerm/internal/services/lighthouse/lighthouse_definition_resource.go

```diff
@@ -5,15 +5,15 @@ import (
 	"log"
 	"time"
 
-	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/subscription/validate"
-
 	"github.com/Azure/azure-sdk-for-go/services/managedservices/mgmt/2019-06-01/managedservices"
+	frsUUID "github.com/gofrs/uuid"
```

Comment:
Can we not alias the package name?

Reply:
there is already a uuid package referenced: "github.com/hashicorp/go-uuid"